src/test/java/fi/helsinki/cs/tmc/core/communication/ExerciseSubmitterTest.java

```diff
@@ -266,7 +267,10 @@ public void submitWithCodeReviewRequest()
         rootFinder.setReturnValue(testPath.toString());
         courseSubmitter.submitWithCodeReviewRequest(testPath, "Help");
         Mockito.verify(urlCommunicator)
-                .makePostWithByteArray(anyString(), any(byte[].class), anyMap(), capture.capture());
+				.makePostWithByteArray(URI.create(anyString()),
```

Comment:
I'd prefer we used `any(URI.class)` unless this does something fundamentally different.